docs/cli-reference/ks_show.md

```diff
@@ -1,14 +1,36 @@
 ## ks show
 
-Show expanded resource definitions
+Show expanded resource definitions for a specific environment.
 
 ### Synopsis
 
 
-Show expanded resource definitions
+Show expanded resource definitions for a specific environment. Jsonnet manifests
+are expanded into their JSON or YAML equivalents (YAML is the default), and parameters
+from the specified environment are used to fill in any components during this process.
```

Comment:
Instead of `and parameters ...fill in any components during this process.`, how about `and parameters get resolved based on their environment-specific value` (or something that emphasizes evaluation).